Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -51,7 +50,7 @@ final class SettingsViewModel: ObservableObject {
             var vehicleToAdd = vehicle
             vehicleToAdd.userID = uid
             
-            try? Firestore
+            _ = try? Firestore
```

Comment:
```suggestion
            try? Firestore
```

Reply:
This would add a warning. What is the point of this change?